doc/user_guide/docs/prerequisites.md

```diff
@@ -0,0 +1,126 @@
+# Prerequisites
+
+This page lists the prerequisites packages that needs to be installed in order to build correctly BioDynaMo.
+BioDynaMo provides also an automated procedure to install all the needed library.
+
+## Ubuntu 16.04, 18.04
+
+### Required Packages
+
+  * **cmake**: Set of tools for automate building, testing of software (for */usr/bin/cmake*);
+  * **make**: Build automation tool (for */usr/bin/make*);
+  * **gcc**: GNU C compiler (for */usr/bin/gcc*);
+  * **g++**: GNU C++ compiler (for */usr/bin/g++*);
+  * **libopenmpi-dev**: Development files for OpenMPI (Open Source Message Passing Interface);
+  * **libomp-dev**: Development files for OpenMP (API for multiprocessor programming);
+  * **libnuma-dev**: Development files for NUMA (simple programming interface to the policy supported by the Linux kernel);
+  * **libtbb-dev**: Development files for TBB (C++ template library developed by Intel for parallel programming);
+  * **freeglut3-dev**: Development files for GLUT (OpenGL Utility Toolkit);
```

Comment:
I think this is only needed by ParaView -> optional
Also change it in the cmake files